spark/src/main/scala/org/apache/spark/sql/delta/TableFeature.scala

```diff
@@ -625,6 +626,18 @@ object ManagedCommitTableFeature
   }
 }
 
+object TypeWideningTableFeature extends ReaderWriterFeature(name = "typeWidening-dev")
```

Comment:
since we are using -dev right now and its not ready for users using it. is it behind the isTesting flag?

Reply:
Yes, it is: https://github.com/databricks/runtime/pull/83562/files#diff-7b78502e6bca8772b4197ce6652fdfa72c4211c15b49e3b03ddfef7199cfcf0aR353